process-engine/bpmn-studio
# ✨Add enum (dropdown) support to property panel

## What did you change?

This PR adds the possibility to add values to a formfield of type enum on usertasks.

## How can others test the changes?

- checkout the branch
- `npm run build`
- `npm run electron-start-dev`
- add an usertask to a diagram 
- add a form to the usertask and choose type enum
- add values => save the diagram => run the process => see a dropdown with the choosable values

## PR-Checklist

Please check the boxes in this list after submitting your PR:
